src/app/components/dog/dog.component.css

```diff
@@ -1,251 +1,262 @@
-    background: white;
-    border-radius:40px;
-    color: rgb(123,123,123);;
-    position: relative;
-    padding-left: 15px;
-    padding-right: 35px;
-    border:1px solid rgba(0,0,0,0.1);
-    outline: none;
-    font-family: 'playfair';
+  width: 190px;
+  height: 40px;
+  background: white;
+  border-radius:40px;
+  color: rgb(123,123,123);;
+  position: relative;
+  padding-left: 15px;
+  padding-right: 35px;
+  border:1px solid rgba(0,0,0,0.1);
+  outline: none;
+  font-family: 'playfair';
+}
+.dog-container .dog-search:focus,
+.dog-container .dog-search:active{
+box-shadow: 0 0 16px #cecece;
 }
 .dog-container .fa-search{
-  position: absolute;
-  top: 15px;
-  right: 45px;
+position: absolute;
+top: 13px;
+right: 45px;
 }
 .dog-container .dog-search-container{
-  position: relative;
+position: relative;
 }
 
 .dog-container .dog-item{
-    width: 200px;
-    height: 75px;
-    float: left;
-    background: white;
-    text-align: center;
-    line-height: 4.2;
-    margin: 0 20px 20px 0;
-    cursor: pointer;
-    box-shadow: 0 5px 15px rgba(0,0,0,0.08);
-    color: rgb(123, 123, 123);
-    font-size: 19px;
-
+  width: 200px;
+  height: 75px;
+  float: left;
+  background: white;
+  text-align: center;
+  line-height: 4.2;
+  margin: 0 20px 20px 0;
+  cursor: pointer;
+  box-shadow: 0 5px 15px rgba(0,0,0,0.08);
+  color: rgb(123, 123, 123);
+  font-size: 19px;
+  text-transform: capitalize;
 }
 .dog-container .dog-item:hover{
-    color: #333;
-    background-color:rgba(0,0,0,0.01);
+  color: #333;
+  background-color:rgba(0,0,0,0.02);
 }
 .dog-container .dog-details{
-    position: absolute;
-    height: auto;
-    min-height: 100vh;
-    width: 100%;
-    background: white;
-    z-index: 20;
-    background: rgba(0,0,0,0.01);
+  position: relative;
+  height: auto;
+  min-height: 100vh;
+  width: 100%;
+  background: white;
+  z-index: 20;
+  background: rgba(0,0,0,0.01);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
 }
 .dog-container .dog-image-container{
-    margin-top: 30px;
-    margin-left: 30px;
-    text-align: center;
-    border: 7px dashed lightgray;
-    overflow: hidden;
-    display: flex;
-    align-items: center;
-    width: 70%;
-    height: 500px;
-    float: left;
+  margin-top: 30px;
+  margin-left: 30px;
+  text-align: center;
+  border: 3px dashed lightgray;
+  overflow: hidden;
+  display: flex;
+  align-items: center;
+  width: 70%;
+  height: 500px;
+  float: left;
 }
 .dog-container .dog-image-container img{
-    display: block;
-    margin-left: auto;
-    margin-right: auto
+  display: block;
+  margin-left: auto;
+  margin-right: auto
 }
 .dog-container .more-dogs{
-  margin-top: 250px;
-  margin-left: 60px;
-  float: left;
-  cursor: pointer;
-  color: rgb(123, 123, 123);
-  font-size: 27px;
-  border: 1px solid rgba(0,0,0,0.03);
-  border-radius: 30px;
-  height: 70px;
-  outline: none;
-  padding: 15px;
-  background: white;
-  font-family: 'playfair'
+/* margin-top: 250px; */
```

Comment:
remove comments

Reply:
I miss comment because if you want you can return back easly